.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish distributions to TestPyPI and PyPI
+
+# Only activate if the v* tag is pushed.
+on:
+  push:
+    tags:
+    - v*
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python distributions to TestPyPI and PyPI
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
```

Comment:
I'm not sure, but is it reasonable to always use python3.7 here?

Reply:
Thank you for your comment. I just followed [the user guide](https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/#checking-out-the-project-and-building-distributions), and I noticed that I didn't check whether we could build the package on GitHub actions.

What do you think if I divide this PR into two?
1. Create distribution packages and store them as artifacts.
2. Publish the package via TestPyPI and PyPI.

Reply:
I agree with your idea of splitting the PR into two parts. I think the two tasks are separated semantically.